unearthed-arcana/2016/20160912.xml

```diff
@@ -0,0 +1,914 @@
+        <rules>
+            <!-- <grant type="Grants" id="ID_INTERNAL_GRANT_MULTICLASS_SPELLCASTING_SLOTS_HALF" requirements="ID_INTERNAL_GRANT_MULTICLASS"/> --> 
+            <!-- <grant type="Grants" name="ID_INTERNAL_GRANTS_AVERAGE_HP_D10" /> -->
+            <grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_LIGHT_ARMOR" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_MEDIUM_ARMOR" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Proficiency" name="ID_PROFICIENCY_ARMOR_PROFICIENCY_SHIELDS" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_SIMPLE_WEAPONS" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Proficiency" name="ID_PROFICIENCY_WEAPON_PROFICIENCY_MARTIAL_WEAPONS" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Proficiency" name="ID_PROFICIENCY_SAVINGTHROW_STRENGTH" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Proficiency" name="ID_PROFICIENCY_SAVINGTHROW_DEXTERITY" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <select type="Proficiency" name="Skill Proficiency (Ranger)" supports="Skill,Ranger" number="3" requirements="!ID_WOTC_UA20160912_MULTICLASS_RANGER" />
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_FAVORED_ENEMY" level="1"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_NATURAL_EXPLORER" level="1"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_FIGHTING_STYLE" level="2"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_SPELLCASTING" level="2"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_RANGER_CONCLAVE" level="3"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_PRIMEVAL_AWARENESS" level="3"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_ASI" level="4"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_GREATER_FAVORED_ENEMY" level="6"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_FLEET_OF_FOOT" level="8"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_HIDE_IN_PLAIN_SIGHT" level="10"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_VANISH" level="14"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_FERAL_SENSES" level="18"/>
+            <grant type="Class Feature" name="ID_WOTC_UA20160912_CLASSFEATURE_RANGER_FOE_SLAYER" level="20"/>
+        </rules>
+        <multiclass id="ID_WOTC_PHB_MULTICLASS_RANGER">
```

Comment:
use the `ID_WOTC_UA20160912_MULTICLASS_RANGER` UA ID here instead of overriding PHB ID